events/views/flow.py

```diff
@@ -461,8 +461,9 @@ def hours_prefill_self(request, id):
     if not event.ccinstances.exists():
         raise PermissionDenied
     if event.hours.filter(user=request.user).exists():
-        messages.add_message(request, messages.ERROR, 'You already have hours for this event.')
+        messages.add_message(request, messages.ERROR, 'You are already checked in to this event.')
```

Comment:
can you test to make sure that this does the expected? It looks like it'll error whenever a member has hours